src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -354,6 +354,12 @@ $externalMargin: 8px;
 		// header is filled with primary or image background
 		filter: none !important;
 		color: var(--color-primary-text) !important;
+
+		// NOTE when migrating to Vue3, while not deprecated, a better solution would be to style the child slotted element
+		// LINK https://stackoverflow.com/a/55368933
+		:deep * {
+			cursor: pointer !important;
+		}
```

Comment:
This is already done by `NcButton`: 
https://github.com/nextcloud-libraries/nextcloud-vue/blob/2bbb1b4eeffbc766341fe4b2d606f83ac7b2b971/src/components/NcButton/NcButton.vue#L728-L731

For some reason, button styles don't work on `ContactsMenu`.